recipes/nbsphinx/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "nbsphinx" %}
+{% set version = "0.2.9" %}
+{% set md5 = "d5a80f5fc9a6f2128b6498c1f41dd079" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docutils
+    - jinja2
+    - nbconvert
+    - nbformat
+    - sphinx
+
+  run:
+    - python
+    - docutils
+    - jinja2
+    - nbconvert
+    - nbformat
+    - sphinx
+test:
+  imports:
+    - sys
```

Comment:
Haha, yes I knew that one would always pass, but I wasn't quite sure how to test [nbsphinx](https://github.com/spatialaudio/nbsphinx/).

I'll put back `import nbsphinx` first, see whether that one passes or not.